src/seedu/addressbook/data/person/Name.java

```diff
@@ -43,6 +43,10 @@ public static boolean isValidName(String test) {
         return Arrays.asList(fullName.split("\\s+"));
     }
 
+    public int compareTo(Name n) {
```

Comment:
Try to make use of java annotation. 
If you use the `@override` annotation, it will warn you in the event where you didn't override properly.